src/state-county-map/StateCountyMap.js

```diff
@@ -99,7 +107,7 @@ export default function StateCountyMap({
             ));
           }}
         </Geographies>
-        {data.map((record) => (
+        {sortedData.map((record) => (
```

Comment:
Interestingly I didn't need to explicitly set a `z-index`.  The charting library seems to respect the order the markers are placed onto the map.

Reply:
Must be setting z-indices under the hood based on order? 🤷